group_vars/location_noki/networks.yml

```diff
@@ -0,0 +1,87 @@
+---
+ipv6_prefix: '2001:bf7:760:300::/56'
+
+# got following prefixes:
+# Router: 10.31.174.0/26
+# --DHCP: 10.31.174.0/27
+# --DHCP: 10.31.174.32/28
+# --MESH: 10.31.174.48/30
+# --UPLK: 10.31.174.52/30
+# --MGMT: 10.31.174.56/29
+
+# Disable noping
+dhcp_no_ping: false
+
+networks:
+  # DHCP with filtering and isolation
+  - vid: 40
+    role: dhcp
+    inbound_filtering: true
+    enforce_client_isolation: true
+    # untagged: true
+    prefix: 10.31.174.0/27
+    ipv6_subprefix: 0
+    assignments:
+      noki-core: 1
+
+  # DHCP without filtering and isolation
+  - vid: 50
+    role: dhcp
+    name: nokihost
+    prefix: 10.31.174.32/28
+    ipv6_subprefix: 50
+    assignments:
+      noki-core: 1
+
+  # MESH - 5 GHz 802.11s
+  - vid: 20
+    role: mesh
+    name: mesh_11s_5ghz
+    prefix: 10.31.174.48/32
+    ipv6_subprefix: -1
+    mesh_ap: noki-core
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # MESH - 2.4 GHz 802.11s
+  - vid: 21
+    role: mesh
+    name: mesh_11s_2ghz
+    prefix: 10.31.174.49/32
+    ipv6_subprefix: -2
+    # make mesh_metric(s) for 2GHz worse than 5GHz
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.8']
+    mesh_ap: noki-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  # MGMT
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.174.56/29
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      # 10.31.174.57/32
+      noki-core: 1
+
+  # UPLK - uplink via Wireguard and local internet connection
+  - vid: 43
+    untagged: true
+    name: uplink
+    role: ext
+    tunnel_wan_ip: 192.168.188.5/24
+    tunnel_wan_gw: 192.168.188.1
+    tunnel_connections: 2
+    tunnel_timeout: 600
+    tunnel_mesh_prefix_ipv4: 10.31.174.52/30
```

Comment:
Nur neugierig, warum /30? Hab meinen Uplinks einfach nur ein /32 aus dem Meshblock gegeben

Reply:
Jeder Tunnel bekommt für OLSR eine eigene Interface IP. Mit zwei Tunnelverbindungen werden 2 IPs benötigt. /32 wäre in diesem Fall also ungenügend. Mit einer Tunnelverbindung wäre /32 natürlich ok.

![image](https://user-images.githubusercontent.com/198902/195837683-a94a046d-68e4-429d-be80-3d1f5a811868.png)